src/javascript/lib/jquery.floatingscroll.js

```diff
@@ -0,0 +1,162 @@
+/*!
+ * jQuery floatingscroll Plugin 2.1.1
```

Comment:
Sorry I did not considered this, so what's the consideration when adding dependency?